Sources/Components/BottomSheet/Transition/BottomSheetPresentationController.swift

```diff
@@ -125,15 +126,18 @@ private extension BottomSheetPresentationController {
     }
 
     @objc func handleTap() {
+        guard stateController.state != .dismissed else { return }
         stateController.state = .dismissed
         gestureController?.velocity = .zero
         presentedViewController.dismiss(animated: true)
+        dismissalDelegate?.bottomSheetDidDismiss()
```

Comment:
Should this be called in the completion handler in the dismiss function instead maybe?